app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+  has_secure_password
+  validates :username, uniqueness: { case_sensitive: false }, presence: true, length: { maximum: 25 }
+  validates :password, presence: true, length: { minimum: 8 }
+
```

Comment:
add email

Reply:
This will be included in the next commit.